packages/config/src/options.ts

```diff
@@ -225,6 +225,13 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     isExperimental: true,
     canUpdateDuringTestTime: false,
     requireRestartOnChange: 'server',
+  }, {
```

Comment:
Our config isn't in the greatest state. As is, e2e testers could set these even if the server didn't honor it. it'd be better to add this to the list of breaking e2e test configuration changes that are lower in this file.

Reply:
Good idea. Also a good chance to solicit feedback from CT users. What do you think of this?

![image](https://user-images.githubusercontent.com/19196536/183801728-d9d2a2c7-daa6-4fd1-8999-721cb359896a.png)